package.json

```diff
@@ -9,7 +9,9 @@
   ],
   "main": "lib/index.js",
   "scripts": {
+    "eslint": "eslint --fix --cache .",
```

Comment:
@jokeyrhyme we'd probably want to run both linters.  Many of the projects at Khan Academy have a `make lint` command so we're trying to stay consistent with that.